src/services/aggregateDonations/aggregateDonations.service.js

```diff
@@ -0,0 +1,38 @@
+module.exports = function aggregateDonations() {
+  const app = this;
+  const donationsService = app.service('donations');
+  const getDonationsInfo = async delegateTypeId => {
+    const query = {
+      _aggregate: [
+        {
+          $match: {
+            delegateTypeId,
+            amountRemaining: { $ne: 0 },
+            intendedProjectId: { $exists: false },
+            isReturn: false,
```

Comment:
These conditions does not return needed data, we should query based on project id. We have different rules based on project type, for campaign this works: 
```
            status: { $ne: Donation.FAILED },
            ownerTypeId: id,
            isReturn: false,
```
It's the same rules in dapp project: https://github.com/Giveth/giveth-dapp/blob/0cd7aff142a5d6ba8f5b9f56e655cd0c134bdb76/src/services/CampaignService.js#L117
Each project type (Milestone, DAC and Campaign) has its own appropriate rules, Find them in giveth-dapp
I guess we can create a uniq rule set applicable for all kind of projects, please checkout and implement if possible.